src/UI/Components/Sidebar.php

```diff
@@ -326,7 +326,7 @@ public function getOutput()
                             return $group;
                         }, []);
 
-                        echo '<div class="column-no-break">';
+                        echo '<div class="overflow-x-scroll max-w-xs" >';
```

Comment:
It looks like the column-no-break class has been removed, however this needs to be there for mobile responsiveness in the sidebar. 
```suggestion
                        echo '<div class="column-no-break overflow-x-scroll max-w-xs" >';
```